.github/workflows/ci.yaml

```diff
@@ -274,7 +279,7 @@ jobs:
         docker network connect kind local-registry
 
         # Document the local registry
-        # https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/generic/1755-communicating-a-local-registry
+        # https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/generic/1755-communicating-a-local-registry wokeignore:rule=master
```

Comment:
Should `wokeignore:rule=master` be commented out here? (`// wokeignore:rule=master`)?

Reply:
ah - actually nevermind. I can see this is reference is made as part of a line that's already a comment, correct?